### PR TITLE
[WIP] fix most errors with new django-rest-framework and bug with exclude

### PR DESCRIPTION
**Pull request by galuszkak**
I will remove [WIP] flag when I finish this. 
For now test will be probably failing.

### CODE REVIEW

**Comment by galuszkak** [comment]
For now we have to fix those problems:
- [ ] `TypeError: ListCreateAPIView() received an invalid keyword u'paginate_by'. as_view only accepts arguments that are already attributes of the class.`
- [ ] `AssertionError: Original widget's attribute was not copied: u'DATE_FORMAT' != '%Y-%m-%d'`
- [ ]  `'%Y-%m-%d' != u'DATE_FORMAT'`

**Comment by andrewsmedina** [comment]
@galuszkak will we drop the support for django 1.6 and 1.7?

**Comment by andrewsmedina** [comment]
@galuszkak the date format problem is because it is hardcoded by floppyforms: https://github.com/andrewsmedina/django-floppyforms/blob/master/floppyforms/widgets.py#L287

I believe that a way to fix it is change tests.

File: tox.ini [hunk 1]
```diff
@@ -12,69 +10,39 @@ setenv=
...
-    Django>=1.6,<1.7
+    Django>=1.9,<1.9.999
     {[testenv]deps}
 
-[testenv:py34-dj1.6.x]
+[testenv:py34-dj1.8.x]
 basepython=python3.4
 deps =
-    Django>=1.6,<1.7
+    Django>=1.8,<1.8.999
```
Comment (auvipy) [inline tox.ini]: this could be 1.9

**Comment by galuszkak** [comment]
closing in favor of #431

**Pull request closed without merging**